docs/dev/Docker.md

```diff
@@ -0,0 +1,104 @@
+# How to build, publish and run a Apache Drill Docker image
+
+## Prerequisites
+
+To build an Apache Drill docker image, you need to have the following software installed on your system to successfully complete a build. 
+  * Java 8
+  * Maven 3.3.1 or greater
+  * Docker 18 or greater
+
+## Confirm settings
+    # java -version
+    java version "1.8.0_161"
+    Java(TM) SE Runtime Environment (build 1.8.0_161-b12)
+    Java HotSpot(TM) 64-Bit Server VM (build 25.161-b12, mixed mode)
+
+    # mvn --version
+    Apache Maven 3.3.1 (cab6659f9874fa96462afef40fcf6bc033d58c1c; 2015-03-13T13:10:27-07:00)
+    
+    # $ docker version
+    Client:
+     Version:      18.03.1-ce
+     API version:  1.37
+     Go version:   go1.9.5
+     Git commit:   9ee9f40
+     Built:        Thu Apr 26 07:13:02 2018
+     OS/Arch:      darwin/amd64
+     Experimental: false
+     Orchestrator: swarm
+
+    Server:
+     Engine:
+      Version:      18.03.1-ce
+      API version:  1.37 (minimum version 1.12)
+      Go version:   go1.9.5
+      Git commit:   9ee9f40
+      Built:        Thu Apr 26 07:22:38 2018
+      OS/Arch:      linux/amd64
+      Experimental: true
+
+## Checkout
+
+    git clone https://github.com/apache/drill.git
```

Comment:
For commands and preformatted tables/lists, might want to use the triple-back-tick format so that the stuff looks like a command and is not HTML-formatted.

Reply:
@paul-rogers, done. Please take a look.